lambdas/src/services/session-request-validator.ts

```diff
@@ -31,11 +36,15 @@ export class SessionRequestValidator {
                 "Session Validation Exception",
                 `Invalid request: JWT validation/verification failed: Redirect uri ${payload.redirect_uri} does not match configuration uri ${expectedRedirectUri}`,
             );
+        } else if (!payload.scope || scope.toLowerCase().indexOf("openid") === -1) {
+            throw new SessionValidationError("Session Validation Exception", "Invalid scope parameter");
+        } else if (!state) {
+            throw new SessionValidationError("Session Validation Exception", "Invalid state parameter");
         }
 
         return payload;
     }
-    private async verifyJwtSignature(jwt: Buffer): Promise<JWTPayload | null> {
+    public async verifyJwtSignature(jwt: Buffer): Promise<JWTPayload | null> {
```

Comment:
This method is `private` no client actually call it. Something is not quite right when you make a method that is private public only so you can test it.

Reply:
Fixed